libs/cardano-ledger-core/src/Cardano/Ledger/Language.hs

```diff
@@ -37,12 +62,40 @@ import Data.Aeson (
   withText,
  )
 import Data.Aeson.Types (toJSONKeyText)
+import qualified Data.ByteString.Base64 as B64 (encode)
+import Data.ByteString.Short (ShortByteString, fromShort)
 import Data.Ix (Ix)
 import Data.Text (Text)
 import Data.Typeable (Typeable)
 import GHC.Generics (Generic)
 import NoThunks.Class (NoThunks)
 
+data Plutus = Plutus
+  { plutusLanguage :: !Language
+  , plutusScript :: !BinaryPlutus
+  }
+  deriving stock (Eq, Ord, Show, Generic)
```

Comment:
Just curious what's the reason for having this  and `BinaryPlutus` in core?   (`PlutusWithContext` has to stay in Alonzo I suppose, because of its dependency on plutus api)

Reply:
Ah, I guess so we can use it in the `Phase2Script`  constructor